include/bluetooth/conn.h

```diff
@@ -147,8 +147,20 @@ struct bt_conn_info {
 
 		struct bt_conn_br_info br;
 	};
+
+	void *conn_ctx;
```

Comment:
Lets not overload the stack structures. This struct contains the right members now. Any applications needs shall be in layered modules with the ease to dispose/replace such module without significant cost on the host stack.